<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMain</class>
 <widget class="QDialog" name="DlgMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Connections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnClearDisconnected">
            <property name="text">
             <string>Clear disconnected</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>864</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblConn">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>State</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IP address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Connected</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Disconnected</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Database</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rx</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tx</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Kick</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Username</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quit reason</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Session id</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActivity">
      <attribute name="title">
       <string>Activity</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="lwActivity"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Config</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="wSettings" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="12" column="1">
             <widget class="QLineEdit" name="leLogPassword"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Airwick database</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="leLogDb"/>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="leLogHost"/>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="rgGenRangom">
                 <property name="text">
                  <string>Random</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rgGenCounter">
                 <property name="text">
                  <string>Counter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>415</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="leUsername"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="leHost"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="leDatabase"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lePassword"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Log database</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0" colspan="2">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>ID genereator style</string>
              </property>
             </widget>
            </item>
            <item row="16" column="0" colspan="2">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leServerPort">
              <property name="text">
               <string>1250</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="leLogUsername"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Database</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Database</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Server port</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="17" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="18" column="1">
             <widget class="QPushButton" name="btnSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QCheckBox" name="chDoNotLog">
              <property name="text">
               <string>Turn off log</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnClearDisconnected</tabstop>
  <tabstop>tblConn</tabstop>
  <tabstop>leServerPort</tabstop>
  <tabstop>leHost</tabstop>
  <tabstop>leDatabase</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leLogHost</tabstop>
  <tabstop>leLogDb</tabstop>
  <tabstop>leLogUsername</tabstop>
  <tabstop>leLogPassword</tabstop>
  <tabstop>rgGenRangom</tabstop>
  <tabstop>rgGenCounter</tabstop>
  <tabstop>chDoNotLog</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>lwActivity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
